ddd-koans/Value Object Pt. 2 & Entity/Introduction/task.html

```diff
@@ -0,0 +1,99 @@
+<html>
+<h3>Feeling a little bit lost</h3>
+<p>After the first lesson about value object you can feel a little bit lost, if not that's good for you :)</p>
+<p>
+    It's hard to simultaneously explain Kotlin, domain-driven design and music, but now, we have hope that basics of
+    value objects are not so scary anymore. Therefore, in the next few task, we will focus on the music.
+</p>
```

Comment:
it just shows that our course is weak :stuck_out_tongue:  - I would just remove this section completely